manifests/repository.pp

```diff
@@ -120,7 +120,7 @@
         }
         $location = "${url}xUbuntu_${osrelease}"
       } else {
-        if $osmajrelease == '10' {
+        if $osmajrelease >= '10' {
```

Comment:
Prefer `versioncmp()` to compare version strings (because regarding string comparison: "10" < "2"):

```suggestion
        if versioncmp($osmajrelease, '10') >= 0 {
```